e2e/utilities/utils.ts

```diff
@@ -23,14 +23,87 @@
  * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
  */
 
-import { browser, protractor, ElementFinder, ExpectedConditions as EC, by, logging } from 'protractor';
+import { browser, protractor, ElementFinder, ExpectedConditions as EC, by, logging, until } from 'protractor';
 import { Logger } from '@alfresco/adf-testing';
-import { BROWSER_WAIT_TIMEOUT, E2E_ROOT_PATH, EXTENSIBILITY_CONFIGS } from '../configs';
+import { BROWSER_WAIT_TIMEOUT, E2E_ROOT_PATH } from '../configs';
 
 const path = require('path');
 const fs = require('fs');
 const StreamZip = require('node-stream-zip');
 
+export async function typeText(element: ElementFinder, text: string) {
+  await element.clear();
+  await element.sendKeys(text);
+}
+
+export async function clearText(element: ElementFinder) {
```

Comment:
maybe the name can be clearWithBackspace ? to express what the method actually does